### ISSUE TITLE
[Bug] Top sites paging does not work with any a11y service

### ISSUE THREAD

**Issue by jonalmeida** [mozilla-mobile/fenix#15125]
## Steps to reproduce
 - Have a two pages of top sites on the homescreen.
 - Use an accessibility provider like TalkBack.
 - Navigate from the first top site page to another on the second page.

### Expected behavior
 - You can navigate to an individual top site on the second screen.

### Actual behavior
 - You cannot.

### Device information

* Android device: Android 11, Pixel 4
* Fenix version: Nightly 14/09/2020

**Comment by eliserichards** [comment]
@jonalmeida do you think this is AC work? Or should I pull it into an eng backlog for Fenix?

**Comment by jonalmeida** [comment]
> @jonalmeida do you think this is AC work? Or should I pull it into an eng backlog for Fenix?

All the UI/frontend work is in Fenix so that would be best. Thanks!

**Comment by mcarare** [comment]
@jonalmeida IMO this works as expected. Changing pages in ViewPager is done the same as navigating pages on device launcher, or scrolling through a list, by using to fingers to swipe.

**Comment by jonalmeida** [comment]
@mcarare on the device launcher, when I navigate with TalkBack and reach the end of the list, the moves to the second page of widgets I have on my screen.

With top sites though, it moves from the end of the first page to collections.

I was expecting the same behaviour as the homescreen, but maybe that isn't typical behaviour when the pager isn't the full screen?

Feel free to close otherwise! 🙂

**Comment by mcarare** [comment]
@jonalmeida that depends on the device / launcher, on some you cannot move on next page in that way. 
And in this case, when you have something below the view pager, IMO is expected to move to collections on single finger swipe, because it's the next item on screen.
